tools/coverage_plugins_package/src/coverage_plugins/jit_plugin.py

```diff
@@ -41,9 +41,13 @@ def dynamic_context(self, frame):
                 filename = getsourcefile(obj)
                 # We don't want to report for filename = None
                 if filename:
-                    sourcelines, starting_lineno = getsourcelines(obj)
-                    line_data = {filename: range(starting_lineno, starting_lineno + len(sourcelines))}
-                    cov_data.add_lines(line_data)
+                    try:
+                        sourcelines, starting_lineno = getsourcelines(obj)
+                    except OSError:
+                        pass
```

Comment:
Could you add some documentation here as of why we have to bypass the error? Either with a comment or a printed warning?

Reply:
Ok I will add a comment because warning would expose our implementation details to user i think

Reply:
done